addons/CLib/Core/Misc/fn_serverInitModCheck.sqf

```diff
@@ -0,0 +1,17 @@
+#include "macros.hpp"
+/*
+    Community Lib - CLib
+
+    Author: joko // Jonas
+
+    Description:
+    Server for Core Module
+
+    Parameter(s):
+    None
+
+    Returns:
+    None
+*/
+GVAR(serverAddons) = [call FUNC(getAllAddons), (getArray (configFile >> QPREFIX + "_IgnoredAddons"))];
```

Comment:
load ignored addons from mission too
remove ignored addons from all addons on serverside before transferring to client